src/Components/Main.js

```diff
@@ -0,0 +1,16 @@
+import HornedBeast from './HornedBeast';
+import { hornedBeasts } from '../data';
+
+function Main() {
+  let beasts = hornedBeasts.map(beast => {
+    return <HornedBeast title={beast.title} description={beast.description} imgUrl={beast.image_url} alt={beast.keyword} />;
```

Comment:
Really awesome work! 👍 

 Once we start to pass a lot of props etc. to other components, indenting them all can prove useful for easier readability. So here, `title, description, imgUrl, alt` could all be indented to rest inside`<HornedBeast />`